# Request
string waypoint_file
---
# Response
geometry_msgs/Point[] map_waypoints
